apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -139,7 +129,7 @@ That’s it! You have now successfully connected a Prisma project to a PostgreSQ
 
 ## Connection pooling with Supabase
 
-If you’re working in a serverless environment (for example Node.js functions hosted on AWS Lambda, Vercel or Netlify Functions), you need to set up [connection pooling](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#serverless-environments-faas) using a tool like [PgBouncer](https://www.pgbouncer.org/). That’s because every function invocation may result in a [new connection to the database](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#the-serverless-challenge). 
+If you’re working in a serverless environment (for example Node.js functions hosted on AWS Lambda, Vercel or Netlify Functions), you need to set up [connection pooling](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#serverless-environments-faas) using a tool like [PgBouncer](https://www.pgbouncer.org/). That’s because every function invocation may result in a [new connection to the database](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#the-serverless-challenge). It is recommended to minimize the number of concurrent connections by setting the `connection_limit` to `1`.
```

Comment:
Connection pooling is not just recommended in a serverless environment, any high request application benefits from it.

The recommendation to set the connection_limit to is still a bad one. This should just be removed unless there is a very specific situation in where it makes sense.